src/components/developed-by/developed-by.js

```diff
@@ -11,7 +11,7 @@ export const DevelopedBy = ({ showSecondLine }) => {
     <>
       <div className="container developed-by">
         <div>
-          <div classNamne="caption">Un proiect în parteneriat cu </div>
+          <div className="caption">Un proiect în parteneriat cu </div>
```

Comment:
This is also my bad, but it was fixed in another PR, must have been re-added on merge conflict